options/options.cc

```diff
@@ -549,6 +549,7 @@ ReadOptions::ReadOptions()
     : snapshot(nullptr),
       iterate_lower_bound(nullptr),
       iterate_upper_bound(nullptr),
+      prefix(nullptr),
```

Comment:
Is this new option use for test only?

Reply:
never mind. I don't know what I was thinking but prefix is needed to skip iterator init before seek..